web/crux-ui/src/components/products/versions/images/edit-image-json.tsx

```diff
@@ -61,7 +61,9 @@ const EditImageJson = (props: EditImageJsonProps) => {
     (err: Error) => {
       throttle(CANCEL_THROTTLE)
 
-      propOnParseError(err)
+      if (propOnParseError) {
```

Comment:
is there a use case where it's legit null?

Reply:
Yes

Reply:
But i didn't check its legitability